.woloxci/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ruby:2.4.0
```

Comment:
We have another dockerfile in the project. Shouldn't we remove that one? I assume the other one was used for Jenkins

Reply:
You're right